src/partisan_peer_connection.erl

```diff
@@ -119,17 +136,63 @@ connect(Address, Port, Options, Timeout) ->
 socket(Conn) ->
     Conn#connection.socket.
 
-%% Internal
+%% @private
 do_connect(Address, Port, Options, Timeout, Transport, Control) ->
    case Transport:connect(Address, Port, Options, Timeout) of
        {ok, Socket} ->
-           {ok, #connection{socket = Socket, transport = Transport, control = Control}};
+           {ok, #connection{socket = Socket, transport = Transport, control = Control, monotonic = false}};
        Error ->
            Error
    end.
 
+%% @private
 tls_enabled() ->
     partisan_config:get(tls).
 
+%% @private
 tls_options() ->
     partisan_config:get(tls_options).
+
+%% @private
+monotonic_now() ->
+    erlang:monotonic_time(millisecond).
+
+%% @private
+send(Transport, Socket, Data) ->
+    %% Update last transmission time.
+    put(last_transmission_time, monotonic_now()),
+
+    %% Transmit the data on the socket.
+    Transport:send(Socket, Data).
+
+%% Determine if we should transmit:
+%%
+%% If theré's another message in the queue, we can skip 
```

Comment:
Minor typo.

Reply:
You making fun of my French keyboard, mate?